packages/suite/src/components/suite/modals/metadata/MetadataProvider/index.tsx

```diff
@@ -1,26 +1,44 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
-import { Modal, Button } from '@trezor/components';
+import { Modal, P, Button, variables, colors } from '@trezor/components';
+
 import { Translation } from '@suite-components';
 import { useActions } from '@suite-hooks';
 import * as metadataActions from '@suite-actions/metadataActions';
 import { Deferred } from '@suite-utils/deferred';
 import { MetadataProviderType } from '@suite-types/metadata';
-import { resolveStaticPath } from '@suite-utils/nextjs';
+
+const { FONT_SIZE, FONT_WEIGHT, SCREEN_SIZE } = variables;
 
 const Buttons = styled.div`
     display: flex;
+    justify-content: center;
     flex-direction: row;
-    justify-content: space-around;
 `;
 
+// todo: can't use button from @trezor/components directly, probably inconsistent design again
+// background-color is not even in components color palette
+
 const StyledButton = styled(Button)`
     padding: 10px;
-    flex-basis: 45%;
+    margin: 0 16px;
+    font-size: ${FONT_SIZE.NORMAL};
+    color: ${colors.BLACK25};
```

Comment:
same as `NEUE_TYPE_DARK_GREY` which is the default color for tertiary button thus this line shouldn't be necessary